src/wad/visitor.rs

```diff
@@ -350,33 +407,33 @@ impl<'a, V: LevelVisitor> LevelWalker<'a, V> {
         self.visitor.visit_volume(self.level.sector_id(sector),
                                   (floor_y,
                                    if is_sky_flat(ceil_tex) {
-                                       from_wad_height(self.height_range.1)
-                                   } else {
-                                       ceil_y
-                                   }),
+                                      from_wad_height(self.height_range.1)
+                                  } else {
```

Comment:
I find this a bit strange, shouldn't it be aligned with the tuple members (i.e. one after the open paren). Is this a bug @nrc ?